crates/solver/src/arguments.rs

```diff
@@ -299,6 +299,23 @@ pub struct Arguments {
     /// value 0.0012 ETH and 0.0016 ETH equivalent.
     #[clap(long, env, default_value = "0")]
     pub solution_comparison_decimal_cutoff: u16,
+
+    #[clap(long, env)]
+    pub s3_instance_upload_region: Option<String>,
+
+    #[clap(long, env)]
+    pub s3_instance_upload_bucket: Option<String>,
+
+    /// Prepended to the auction id to form the final instance filename on S3. Something like
+    /// "staging/mainnet/quasimodo/". Should end with `/` if intended to be a folder.
+    #[clap(long, env)]
+    pub s3_instance_upload_filename_prefix: Option<String>,
+
+    #[clap(long, env)]
+    pub s3_instance_upload_access_key_id: Option<String>,
+
+    #[clap(long, env)]
+    pub s3_instance_upload_secret_access_key: Option<String>,
```

Comment:
Recently we started to group related arguments into a separate argument struct which overall seemed pretty nice. This could be done here as well.

Reply:
I don't strongly feel either way but I slightly prefer this. The arguments are already grouped by having a similar name and being next to eachother. I don't see a benefit from a struct except when that allows us to reuse the same arguments struct.

Reply:
Having a struct for this would also give us a nice place to put the verification logic that is currently free floating in `main()`.

Reply:
I like that argument. Will do.